src/pages/index.html

```diff
@@ -0,0 +1,504 @@
+                    >
+                        <img src="assets/images/data-bases.svg" alt="" />
+                        <div class="number-span">
+                            <span>2</span>
+                        </div>
+                    </div>
+                    <h4>Better Data Sources</h4>
+                    <p>
+                        Ensuring data accuracy is crucial for making informed
+                        decisions and ensuring the reliability.
+                    </p>
+                </div>
+            </div>
+            <div class="col-12 col-lg-1">
+                <div
+                    class="arrow-circle d-flex justify-content-center align-items-center"
+                >
+                <svg width="142" height="16" viewBox="0 0 142 16" fill="none" xmlns="http://www.w3.org/2000/svg">
+                    <path d="M140.72 8.70711C141.111 8.31658 141.111 7.68342 140.72 7.29289L134.356 0.928932C133.966 0.538408 133.333 0.538408 132.942 0.928932C132.552 1.31946 132.552 1.95262 132.942 2.34315L138.599 8L132.942 13.6569C132.552 14.0474 132.552 14.6805 132.942 15.0711C133.333 15.4616 133.966 15.4616 134.356 15.0711L140.72 8.70711ZM0.0131836 9H140.013V7H0.0131836V9Z" fill="#6F4BE5"/>
+                    </svg>
+                    
+                </div>
+            </div>
+            <div class="col-12 col-lg-3">
+                <div
+                    class="stract-works-card d-flex flex-column justify-content-center align-items-center"
+                >
+                    <div
+                        class="work-circle work-circle-black position-relative"
+                    >
+                        <img src="assets/images/Analytics-1.svg" alt="" />
+                        <div class="number-span">
+                            <span>3</span>
+                        </div>
+                    </div>
+                    <h4>Business Analytics</h4>
+                    <p>
+                        Ensuring data accuracy is crucial for making informed
+                        decisions and ensuring the reliability.
+                    </p>
+                </div>
+            </div>
+        </div>
+    </div>
+</section>
+
+<section class="ready-start">
+    <div class="container">
+        <div class="row justify-content-center">
+            <div class="col-12 col-lg-6">
+                <div
+                    class="ready-start-section d-flex flex-column justify-content-center align-items-center"
+                >
+                    <h2>Ready to Get Started?</h2>
+                    <p>
+                        Data analysis software is a type of software tool used
+                        for data analysis and reporting. It is designed to help
+                        businesses, organizations.
+                    </p>
+                    <button>Contact us</button>
+                </div>
+            </div>
+        </div>
+    </div>
+</section>
```

Comment:
@shaban011147 @M-Khairy-B Not like the design

Reply:
Done